documentation/docs/tutorials/openfga.mdx

```diff
@@ -0,0 +1,105 @@
+# OpenFGA and OPAL
+
+[OpenFGA](https://openfga.dev) is an open-source authorization engine implementing the Zanzibar authorization model created by Google. OPAL provides realtime synchronization and management capabilities for OpenFGA, allowing you to keep your authorization data and relationships up-to-date across distributed OpenFGA instances.
+
+## Quick Start
+
+OPAL can run OpenFGA as a policy engine instead of OPA. To launch an example configuration with Docker-Compose:
+
+```bash
+git clone https://github.com/permitio/opal.git
+cd opal
+docker-compose -f docker/docker-compose-example-openfga.yml up -d
+```
+
+## Interacting with OpenFGA
+
+The OpenFGA API will be available at [http://localhost:8080](http://localhost:8080). Here are some key endpoints and operations:
+
+### Check Current State
+- View relationship tuples: `GET /stores/{store_id}/read`
+- List authorization models: `GET /stores/{store_id}/authorization-models`
+
+### Authorization Checks
+To check permissions, send a POST request to `/stores/{store_id}/check`:
+```json
+{
+  "tuple_key": {
+    "user": "user:anne",
+    "relation": "reader",
+    "object": "document:budget"
+  }
+}
+```
+
+### Setting Up Authorization Models
+Create or update an authorization model with PUT to `/stores/{store_id}/authorization-models`:
+```json
+{
+  "schema_version": "1.1",
+  "type_definitions": [
+    {
+      "type": "user"
+    },
+    {
+      "type": "document",
+      "relations": {
+        "reader": {
+          "this": {}
+        },
+        "writer": {
+          "this": {}
+        }
+      }
+    }
+  ]
+}
+```
+
+### Writing Relationship Tuples
+Add new relationships with a POST to `/stores/{store_id}/write`:
```

Comment:
Also here, the most important point, is that you can connect data via data fetchers, better to use this example here